### DISCUSSION TITLE
Improve admin only configuration documentation

### DISCUSSION THREAD

**Discussion by brummelte** [renovatebot/renovate#22195] [Suggest an Idea]
In #7584 I was told that I was trying to add an admin only configuration to the repository config.

Reading the documentation (https://docs.renovatebot.com/self-hosted-configuration) does not really make it clear where that config file would be.

Searching the documentation website I found this (https://docs.renovatebot.com/self-hosting/) page. But I don't see it linked in the sidebar. Or another very good place to link to it would be at the top of the `self-hosted-configuration` documentation page.

This (https://docs.renovatebot.com/self-hosting/#configuration) section makes it really clear and would probably be good at to the top of `self-hosted-configuration` too.

It is easy to figure out, but maybe it would also be good to document that the CLI params are just dasherized Self-Hosted Configuration Options. Or alternatively explicitly listing it on every configuration e.g.:
![image](https://user-images.githubusercontent.com/7269775/97462170-bdec9f80-193e-11eb-8bad-5ab4e20a7f1e.png)

Instead of:
![image](https://user-images.githubusercontent.com/7269775/97462245-cd6be880-193e-11eb-8e5c-805a41aeaaf6.png)

**Comment by HonkingGoose** [comment]
Is there a reason we didn't put the `Self-Hosting Examples` [^examples] page on the sidebar?

We could put the self-hosting examples in our sidebar like this:

- Home
  - Getting Started
  - Troubleshooting
  - Configuration
    - Self-hosted
    - Self-Hosting examples [^examples]
    - Repository
    - Preset
  - Ignore everything else not relevant

[^examples]: https://docs.renovatebot.com/examples/self-hosting/

**Comment by rarkins** [comment]
Currently the Configuration Section is auto-generated based on our code. I would think it fits in better under here:

![image](https://user-images.githubusercontent.com/6311784/139040113-7b03c81e-d5ed-4607-b585-38d2e3c26c8d.png)

**Comment by HonkingGoose** [comment]
- Home
  - Getting Started
    - Use Cases
    - Running Renovate
      - Self-Hosting examples <--- Insert page here???
  - Ignore everything else not relevant
  
Is this what you want then @rarkins?

**Comment by rarkins** [comment]
It could be in parallel under Running Renovate (before installing & onboarding)

**Comment by HonkingGoose** [comment]
Okay, got it! 😄 

I'll go open a PR on the documentation repository to get this in the sidebar in the correct location then.

**Comment by HonkingGoose** [comment]
I've opened a PR to add the page to the sidebar, but I think we still need to fix/improve some more stuff:

> It is easy to figure out, but maybe it would also be good to document that the CLI params are just dasherized Self-Hosted Configuration Options. Or alternatively explicitly listing it on every configuration e.g.: ![image](https://user-images.githubusercontent.com/7269775/97462170-bdec9f80-193e-11eb-8bad-5ab4e20a7f1e.png)
> 
> Instead of: ![image](https://user-images.githubusercontent.com/7269775/97462245-cd6be880-193e-11eb-8e5c-805a41aeaaf6.png)

Not sure how we could do this part of the issue? Any ideas?
